exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/QueryResultHandler.java

```diff
@@ -351,20 +352,5 @@ public void success(QueryId queryId, ByteBuf buf) {
         }
       }
     }
-
-    @Override
-    public void interrupted(final InterruptedException ex) {
```

Comment:
how about [this](https://github.com/apache/drill/blob/master/exec/rpc/src/main/java/org/apache/drill/exec/rpc/RemoteConnection.java#L74) ?